sources/core/Stride.Core.Reflection/TypeDescriptors/OldCollectionDescriptor.cs

```diff
@@ -243,16 +243,6 @@ public override int GetCollectionCount(object collection)
             return collection == null || GetCollectionCountFunction == null ? -1 : GetCollectionCountFunction(collection);
         }
 
-        /// <summary>
-        /// Determines whether the specified type is collection.
-        /// </summary>
-        /// <param name="type">The type.</param>
-        /// <returns><c>true</c> if the specified type is collection; otherwise, <c>false</c>.</returns>
-        public static bool IsCollection(Type type)
```

Comment:
This hides the base class implementation so should be 'new' but it turns out this is a duplicate of that code so easily removed.